### PR TITLE
Nds health check fix

### PR DESCRIPTION
**Pull request by neildsouth**
### Description

the health check wasn't making an Admin account alias but was checking to see if it was present. and so failing all the time.

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] All tests passed locally.
- [ ] [Documentation comments](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments) included/updated.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -45,6 +45,8 @@ jobs:
     - uses: actions/setup-dotnet@v3
       with:
         dotnet-version: "6.0.x"
+    - name: Enable Homebrew
```
Comment (mocsharp) [inline .github/workflows/ci.yml:48]: dup? same as line 51-52?

File: src/Plugins/AWSS3/Monai.Deploy.Storage.AWSS3.csproj [hunk 2]
```diff
@@ -16,36 +16,36 @@
...
 <Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
-    <LangVersion>latest</LangVersion>
-    <RootNamespace>Monai.Deploy.Storage.AWSS3</RootNamespace>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
-    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
-  </PropertyGroup>
+	<PropertyGroup>
```
Comment (mocsharp) [inline src/Plugins/AWSS3/Monai.Deploy.Storage.AWSS3.csproj:19]: nit: untabify?

File: src/Plugins/MinIO/MinIoAdminHealthCheck.cs [hunk 3]
```diff
@@ -36,8 +36,11 @@ public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context
         {
             try
             {
-                var hasConnection = await _storageAdminService.HasConnectionAsync();
-                var connectionResult = await _storageAdminService.GetConnectionAsync();
+                var hasConnection = await _storageAdminService.HasConnectionAsync().ConfigureAwait(false);
+                if (hasConnection is false)
+                { await _storageAdminService.SetConnectionAsync().ConfigureAwait(false); }
```
Comment (mocsharp) [inline src/Plugins/MinIO/MinIoAdminHealthCheck.cs:41]: nit: formatting

**Review by mocsharp — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (neildsouth) [reply .github/workflows/ci.yml:48]: I didnt even change this file !
It must have got bought in with a merge !

File: src/Plugins/AWSS3/Monai.Deploy.Storage.AWSS3.csproj (hunk 2 above)
Comment (neildsouth) [reply src/Plugins/AWSS3/Monai.Deploy.Storage.AWSS3.csproj:19]: again merge in issue !

File: src/Plugins/MinIO/MinIoAdminHealthCheck.cs (hunk 3 above)
Comment (neildsouth) [reply src/Plugins/MinIO/MinIoAdminHealthCheck.cs:41]: will check

**Pull request merged**